packages/cli/src/commands/setup/i18n/i18n.js

```diff
@@ -0,0 +1,90 @@
+import fs from 'fs'
+import path from 'path'
+
+import Listr from 'listr'
+import execa from 'execa'
+import chalk from 'chalk'
+
+import c from 'src/lib/colors'
+import { getPaths, writeFile } from 'src/lib'
+
+export const command = 'i18n'
+export const description = 'Setup i18n'
+export const builder = (yargs) => {
+  yargs.option('force', {
+    alias: 'f',
+    default: false,
+    description: 'Overwrite existing configuration',
+    type: 'boolean',
+  })
+}
+
+const INDEX_JS_PATH = path.join(getPaths().web.src, 'index.js')
```

Comment:
We should move this inside of "handler" since it'll run when the command is imported, and we only want it to run when the command is run. 

This might be a `.tsx` file too; so maybe it's better to resolve this in `getPaths()` like I've done for routes over here:

https://github.com/redwoodjs/redwood/blob/b3cc624a1a8daf6c8aee74f61dd7b3ddf9a76bb3/packages/internal/src/paths.ts#L112

Maybe by adding `getPaths().web.index`

Reply:
Fixed